apps/nrm/models/write-saved-report-data.js

```diff
@@ -44,10 +44,14 @@ const sendDataToBeStored = async(data) => {
   const formData = getSavedFormData(data);
   const visitedPages = data.steps.toString();
 
+  const dateObject = new Date();
+  const currentDateTime = dateObject.toISOString();
```

Comment:
I reckon two declarations are overkill here, `new Date().toISOString()` would be sufficient.

Reply:
cheers Douglas